src/App.js

```diff
@@ -0,0 +1,19 @@
+import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Form from './Component/componentStling/Form';
+import Navbar from './Component/componentStling/navbar';
+import Categories from './Component/categories';
+
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
```

Comment:
[REQUIRED] You should have a Books component which has a BookList component. The Book component should render a list of books and have a button for Removing books. You should also have a NewBookForm component which receives inputs for the title and author of a book and also has a submit button. You seem to have the code for some of these. functionalities but they are not structured in the way described above which is a requirement